components/ILIAS/Forum/classes/Drafts/ForumDraftsTable.php

```diff
@@ -97,7 +98,10 @@ public function initRecords(): void
                 } else {
                     $this->records[$draft_id]['draft'] = $draft['subject'];
                 }
-                $this->records[$draft_id]['edited_on'] = new ilDateTime($draft['post_update'], IL_CAL_DATETIME);
+                $this->records[$draft_id]['edited_on'] = new DateTimeImmutable(
+                    $draft['post_update'],
+                    new DateTimeZone('UTC')
```

Comment:
Please use the `Clock` data type provided by the `ILIAS\Data\Factory`. Furthermore, I am not sure if `UTC` ist the correct timezone. It is most probably the system/default timezone.

Reply:
But Clock just gives me the current time clock()->system()->now()
It doesn't help with updated timestamp, or am I wrong?

Reply:
You are right, **my** bad. I thought you can retrieve a `now` instance based on a given timezone (see: `\ILIAS\Data\Clock\ClockFactory::local`) and set the date/time based on `$draft['post_update']`. But this does not make any sense.

Nevertheless, `UTC` is wrong here, since it is a `date(...)`-base string (see: `\ilForumPostDraft::saveDraft`) based on the system timezone (I tried to introduce UTC based unix timestamps here https://github.com/ILIAS-eLearning/ILIAS/pull/2420 , but I decided to close the PR because of JF discussions we had regarding this forum PR and similar discussions for `Mail` here  https://github.com/ILIAS-eLearning/ILIAS/pull/2413 / https://mantis.ilias.de/view.php?id=26951)